source/includes/_resource.videos.md

```diff
@@ -141,65 +141,127 @@ $ curl -X POST "https://api.vhx.tv/videos" \
       </td>
       <td>An accessible master video file per our compression settings. To grant us permission to download the video source_url from your S3 bucket, see our <a href="https://gist.github.com/ksheurs/d57e4d1857c7ef9465fb" target="_blank">S3 policy</a>.</p></td>
```

Comment:
I don't think we've verified this actually works. Should we remove `To grant us permission to download the video source_url from your S3 bucket, see our <a href="https://gist.github.com/ksheurs/d57e4d1857c7ef9465fb" target="_blank">S3 policy</a>.`

Reply:
Do we know it doesn't work? If we don't know, I feel like we should just leave it since it is currently available. It is possible someone is already using it so removing it feels a bit weird for that user potentially.

Reply:
definitely worth doing a test. but that's fine, let's leave it in. also wish it didn't belong to a vhx-er who hasn't worked here for quite some time.

Reply:
Good point about it belonging to someone who doesn't work here anymore. That is a bit of a dependency that he could take it down at any time which would break the document. I'm kind of rethinking removing it. We could just download the contents and give it to Enterprise support so they can provide it if someone asks but we don't need to keep it public.

Reply:
Yes. also we're not removing the functionality, just an explicit reference to it. I imagine anyone who has this set up isn't checking the API docs regularly

Reply:
Cool, yea I'm gonna remove it. I saved a note for myself with the link and obviously it'll be available in GitHub should we somehow lose it.